_careers/our-hiring-process.md

```diff
@@ -14,7 +14,6 @@ Whilst our specific hiring process varies between roles, we hold a common philos
 **You will have full knowledge of what we’re looking for and how your assessment went.** _You should have what you need to prepare, so that you can show up and interview at your best._
 
 *   We share details on what to expect for every interview stage and what we are looking for.
```

Comment:
Suggestion to merge the 2 sentences to this:

```suggestion
**You will have full knowledge of what we’re looking for and how your assessment went.** We share details on what to expect for every interview stage and what we are looking for. _You will have what you need to prepare, so that you can show up and interview at your best._

```